docs/tutorials/go-built-in.md

```diff
@@ -700,7 +700,7 @@ The additional logic at the end of the file allows the program to catch SIGTERM.
 
 Our application is almost ready to run, but first we'll need to populate the CometBFT configuration files.
 The following command will create a `cometbft-home` directory in your project and add a basic set of configuration files in `cometbft-home/config/`.
-For more information on what these files contain see [the configuration documentation](https://github.com/cometbft/cometbft/blob/v0.37.x/docs/core/configuration.md).
+For more information on what these files contain see [the configuration documentation](../core/configuration.md).
```

Comment:
The version used in the tutorial is the 0.37.0. That is why a full link was used here.

Reply:
thanks for the context @cason, I'll revert this change. We can figure out in the future a better way to link that when we do the docs revamp for example.